web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -579,7 +579,7 @@ class AppConfig extends Component<Props, State> {
     }
 
     return (
-      <div className="flex flex-column u-padding--20 alignItems--center">
+      <div className="flex flex-column u-paddingLeft--20 u-paddingBottom--20 u-paddingRight--20 alignItems--center">
```

Comment:
remove the top padding from the inner row containing the nav and the form. This keeps the side-nav from scrolling behind the top nav